js/cd-dropdown.js

```diff
@@ -257,21 +257,28 @@
     setToggable: function (element) {
       var toggler = element.previousElementSibling;
 
-      // Skip if the toggler has already been created.
-      if (toggler && toggler.hasAttribute('data-cd-toggler')) {
-        return;
+      // Skip if the toggler is not a button or has already been processed.
+      if (toggler) {
+        // Togglers should be buttons to avoid mis-processing elements
+        // appearing before the toggable element. There is still a risk of
+        // mis-processint if, for whatever reason, there is a button which is
+        // not the toggler before the toggable element.
+        if (toggler.nodeName !== 'BUTTON' || ) {
```

Comment:
The hanging `||` -  should there be an OR condition? Or should this be removed?
Removing it makes everything work for me

Reply:
Sorry, I split the 2 conditions to better explain their purpose and missed that... Thanks for spotting the problem.